### ISSUE TITLE
Fix Chat Browser Tests

### ISSUE THREAD

**Issue by ebebbington** [ebebbington/copytube#93]
For some reason, dusk isn't playing ball inside the `ChatPageTest.php`, both clients dont seem too be connected. For example, when both users join, the call button text doesn't seem to change which makes me think there's some problem when using dusk here or maybe the sockets are seeing the clients as connections

**Comment by ebebbington** [comment]
Can confirm the socket isnt receiving connections

**Comment by ebebbington** [comment]
Getting the following in `tests/Browser/console/somelog.log`:

```
{
        "level": "SEVERE",
        "message": "http:\/\/copytube_nginx:9002\/js\/app.js 29857 WebSocket connection to 'ws:\/\/127.0.0.1:9008\/realtime' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED",
        "source": "javascript",
        "timestamp": 1586651752890
    },
    {
        "level": "SEVERE",
        "message": "http:\/\/copytube_nginx:9002\/js\/app.js 29870:12 \"Realtime encountered an error\"",
        "source": "console-api",
        "timestamp": 1586651752893
    },
```

**Comment by ebebbington** [comment]
Fixed partially in #64, updated the url to support dusk, but running dusk doesnt seem to be able toa ccess `navigator`
